.github/workflows/test.yml

```diff
@@ -46,6 +45,17 @@ jobs:
       run: |
         poetry install --no-root
 
+    - name: Set PYTHONPATH
+      run: |
+        echo "PYTHONPATH=${GITHUB_WORKSPACE}" >> $GITHUB_ENV
```

Comment:
why is this needed now?

Reply:
It was needed for testing, but now that the tests are mocked for `list-jobs`, I don't think this is needed anymore. Removed in  b24788b